<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyClippingSettings</class>
 <widget class="QWidget" name="PropertyClippingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>797</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="boxSizeGroupBox">
     <property name="title">
      <string>Box default size</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="SizeLayout">
        <item>
         <layout class="QVBoxLayout" name="XLayout">
          <item>
           <widget class="QLabel" name="XLabel">
            <property name="text">
             <string>Box X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleEdit" name="XInfield">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="YLayout">
          <item>
           <widget class="QLabel" name="YLabel">
            <property name="text">
             <string>Box Y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleEdit" name="YInfield">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ZLayout">
          <item>
           <widget class="QLabel" name="ZLabel">
            <property name="text">
             <string>Box Z</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleEdit" name="ZInfield">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="PropertyDuplicationSettings" name="duplicationWidget" native="true"/>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="positionSettingsGroupBox">
     <property name="title">
      <string>Position settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <widget class="QRadioButton" name="TopFaceRadio">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Box top face on clicked point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="CenterRadio">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Box center on clicked point</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="BottomFaceRadio">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Box Bottom face on clicked point</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="alignementSettingsGroupBox">
     <property name="title">
      <string>Alignement settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="2" colspan="3">
       <layout class="QHBoxLayout" name="SizeLayout_5">
        <item>
         <layout class="QVBoxLayout" name="XLayout_5">
          <item>
           <widget class="QDoubleEdit" name="XP2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="YLayout_5">
          <item>
           <widget class="QDoubleEdit" name="YP2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ZLayout_5">
          <item>
           <widget class="QDoubleEdit" name="ZP2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="point2Button">
        <property name="text">
         <string>Pick point 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2" colspan="3">
       <layout class="QHBoxLayout" name="SizeLayout_4">
        <item>
         <layout class="QVBoxLayout" name="XLayout_4">
          <item>
           <widget class="QLabel" name="XLabel_4">
            <property name="text">
             <string>X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleEdit" name="XP1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="YLayout_4">
          <item>
           <widget class="QLabel" name="YLabel_4">
            <property name="text">
             <string>Y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleEdit" name="YP1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ZLayout_4">
          <item>
           <widget class="QLabel" name="ZLabel_4">
            <property name="text">
             <string>Z</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleEdit" name="ZP1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="userRadioButton">
        <property name="text">
         <string>User axis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="point1Button">
        <property name="text">
         <string>Pick point 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="pointsRadioButton">
        <property name="text">
         <string>2 points</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="projectAxisRadioButton">
        <property name="text">
         <string>Project axis</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
  <customwidget>
   <class>PropertyDuplicationSettings</class>
   <extends>QWidget</extends>
   <header>propertyduplicationsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CenterRadio</tabstop>
  <tabstop>BottomFaceRadio</tabstop>
  <tabstop>TopFaceRadio</tabstop>
  <tabstop>XInfield</tabstop>
  <tabstop>YInfield</tabstop>
  <tabstop>ZInfield</tabstop>
  <tabstop>projectAxisRadioButton</tabstop>
  <tabstop>userRadioButton</tabstop>
  <tabstop>pointsRadioButton</tabstop>
  <tabstop>point1Button</tabstop>
  <tabstop>XP1</tabstop>
  <tabstop>YP1</tabstop>
  <tabstop>ZP1</tabstop>
  <tabstop>point2Button</tabstop>
  <tabstop>XP2</tabstop>
  <tabstop>YP2</tabstop>
  <tabstop>ZP2</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
